cmd/cli/plugin/managementcluster/delete.go

```diff
@@ -45,6 +46,11 @@ func init() {
 }
 
 func runDeleteRegion(server *v1alpha1.Server) error {
+	err := tkgctl.SetCompatibilityFileBasedOnEdition()
+	if err != nil {
+		log.V(3).Infof("%v", err.Error())
+	}
+
```

Comment:
I don't know when this compatibility file is needed, so I can't review if all cases are covered.
However I notice that it is being set when deleting a management cluster but not a normal cluster.  Is that ok?

Reply:
Yeah. Good observation. We need BoM file when deleting a management cluster because we need to spin-up cleanup kind cluster and deploy providers and etc to the cleanup kind cluster. But to delete the workload cluster we do not need to use BoM file and hence it is not been added for workload cluster deletion.